integration_test/sql/migration.exs

```diff
@@ -111,6 +111,29 @@ defmodule Ecto.Integration.MigrationTest do
     end
   end
 
+  defmodule RenameColumnMigration do
+    use Ecto.Migration
+
+    def up do
+      create table(:rename_col_migration) do
+        add :to_be_renamed_inside_alter, :integer
+        add :to_be_renamed_outside_alter, :integer
+      end
+
+      alter table(:rename_col_migration) do
```

Comment:
Let's suppose rename only outside of alter table. I would suggest:

    rename table(:rename_cel_migration), :old_column, to: :new_column